addons/purchase/models/product.py

```diff
@@ -48,6 +49,14 @@ def action_view_po(self):
         }
         return action
 
+    @api.onchange('company_id')
+    def _onchange_company_id(self):
+        if self.company_id:
+            po_company_ids = self.env['purchase.order.line'].search([('product_id', 'in', self.product_variant_ids.ids), ('company_id', '!=', self.company_id.id)])
```

Comment:
Don't we need here the term `('company_id', '!=', False)`?

Reply:
company id is required for PO lines, should I add the `!= False` anyway?